src/systems.rs

```diff
@@ -358,7 +358,7 @@ pub fn process_input_system(
     }
 
     for mut context in context_params.contexts.iter_mut() {
-        context.egui_input.predicted_dt = time.raw_delta_seconds();
+        context.egui_input.predicted_dt = time.delta_seconds();
```

Comment:
Would that change have unwanted effects? It was `raw_delta_seconds` , meaning it should use `Time<Real>`, and yet I think this system uses virtual time

Reply:
no idea

Reply:
Function signature should be updated to use Time<Real>

Reply:
ok